apps/storefront/src/components/HeadlessController.tsx

```diff
@@ -186,6 +191,20 @@ export default function HeadlessController({ setOpenPage }: HeadlessControllerPr
             storeDispatch(setB2BToken(b2bStorefrontJWTToken));
             await getCurrentCustomerInfo(b2bStorefrontJWTToken);
           },
+          logout: async () => {
+            try {
+              if (isAgenting) {
+                await endMasquerade();
+              }
+            } catch (e) {
+              b2bLogger.error(e);
+            } finally {
+              // SUP-1282 Clear sessionStorage to allow visitors to display the checkout page
```

Comment:
Let's add the ticket info to the commit name & branch name, that way it's available in git history but we don't need to leave this in code.